### DISCUSSION TITLE
How to add initial data in migration?

### DISCUSSION THREAD

**Discussion by golubvladimir** [prisma/prisma#7357] [Q&A] [answered]
I want to add initial data in table in migration?

**Comment by ryands17** [comment, marked as answer]
@golubvladimir 👋 

Currently we do not have a direct way of adding initial data in migrations, but you can use [seeding](https://www.prisma.io/docs/guides/database/seed-database) and run the seed script to add initial data after your migration.

I would suggest following this request for more info :)
https://github.com/prisma/prisma/issues/6345

**Reply by aliaksandrIt** [reply]
any udates here? @ryands17

**Comment by hyunwoona** [comment]
@ryands17 Are there any new developments on this front?

**Reply by ryands17** [reply]
Not anything at the moment unfortunately.

**Reply by palashCItobuz** [reply]
@ryands17 Are there ways to run different seed files at different points? So a seed1.ts, seed2.ts etc ?

**Reply by ryands17** [reply]
You can specify only a single seed file to be run at the moment. The files can be different but only a single file can be run using the `prisma db seed` command.

**Comment by Softjey** [comment]
Hi everyone! 👋

I’ve developed a small library to help address this problem: [Prisma Data Migrations](https://www.npmjs.com/package/prisma-data-migrations?activeTab=readme). This library enhances Prisma’s migration system by enabling the use of post-migration scripts for data migrations alongside schema changes.

You can find the repository here: [GitHub Repository](https://github.com/Softjey/prisma-data-migrations).

Feel free to open issues, report bugs, or suggest improvements. I’m also open to pull requests if you’d like to contribute to the project!

Looking forward to your feedback! 🚀

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
